### ISSUE TITLE
[v1] Another option to use custom icons set

### ISSUE THREAD

**Issue by anhmv** [quasarframework/quasar#3203]
Could you guys provide another option for developers can use their own icons set which have difference prefix.

https://github.com/quasarframework/quasar/blob/f8381624635f046c72f5ad1d596763ec7a85e182/src/components/icon/QIcon.js#L41

**Comment by godxavia** [comment]
I've got the same issue when i use ionicon set.

In QInput text field when error show it's still call up the material icon so
the Error icon becomes just a text.

**Comment by rstoenescu** [comment]
@godxavia Fixed the QInput error icon and also found QBtnDropdown had same problem. Fix will be available in quasar beta.2.

@anhmv You haven't specified if this is 0.17 or 1.0. If 0.17, are you referring to [this](https://quasar-framework.org/components/icons.html#Quasar-Using-an-Icon-Set)? If 1.0, are you referring to setting a [Quasar Icon Set](https://v1.quasar-framework.org/options/quasar-icon-sets)? If I misunderstood, pls feel free to reopen. Thank you.

**Comment by anhmv** [comment]
@rstoenescu Sorry, I did not explain enough. I just want to use another icon set, that was not supported by Quasar, not Material Icons, Font Awesome, Ionicons, MDI and Eva Icons. I have looked on quasar-extra repo, and implemented another icon, that has a different prefix, ex: `ti-`. But looks like the `QIcon` component only support icon starts with `fa-` `bt ` `eva` `ion-` `mdi`, when the prefix does not match, quasar auto use material icons by default. And I cannot use my own icon set without edit component `QIcon`, to support new prefix.
